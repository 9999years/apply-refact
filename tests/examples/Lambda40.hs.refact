[("tests/examples/Lambda40.hs:1:12-24: Warning: Avoid lambda\nFound:\n  \\ x -> (x `f`)\nPerhaps:\n  f\n",[])]
