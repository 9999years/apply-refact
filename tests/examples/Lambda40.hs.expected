foo = bar (\x -> (x `f`))
